### PR TITLE
feat(WEG-39): Records List

### PR DESCRIPTION
**Pull request by vogelino**
This PR adds:
- A Labels Context to the pages needing the labels for easy access
- Reduces the size of the records array for a smaller hydration (see `MinimalRecordType`)
- Adds labels and target audience to the `FacilityListItem` component
- Adds hover and focus styles to the `FacilityListItem` component

Missing because of other dependencies:
- The labels do not show as active (red) when a filter is active (as it isn't implemented yet)
- The MapLayout isn't yet implemented (To avoid conflicts with @dnsos)

### CODE REVIEW

File: src/lib/createGeojsonStructure/index.ts [hunk 1]
```diff
@@ -31,12 +31,9 @@ export const createGeoJsonStructure = (
         geometry: {
           type: 'Point',
           // It's curious that the Grist API returns the field long2, while actually in the spreadsheet the column is called long:
```
Comment (dnsos) [inline src/lib/createGeojsonStructure/index.ts:33]: This comment is now unnecessary.

File: src/components/FacilityListItem.tsx [hunk 2]
```diff
@@ -1,39 +1,65 @@
...
+    latitude,
+    longitude,
   })
-  const isOpened = useIsFacilityOpened(fields)
+  const isOpened = useIsFacilityOpened(record)
+  const labelsWithData = useLabels()
+  const recordLabels = labels
+    .map((lId) => labelsWithData.find((l) => l.id === lId))
+    .filter(Boolean) as GristLabelType[]
+  const semanticLabels = recordLabels.filter(
```
Comment (dnsos) [inline src/components/FacilityListItem.tsx:30]: I wonder if we can name this a bit clearer? How about `topicLabels` or `issueLabels`?

**Review by dnsos — APPROVED** [review]
Very clear, good work! I have two small comments, but feel free to merge already!

File: src/lib/createGeojsonStructure/index.ts (hunk 1 above)
Comment (vogelino) [reply src/lib/createGeojsonStructure/index.ts:33]: Why? It is still called long2 right?

File: src/components/FacilityListItem.tsx (hunk 2 above)
Comment (vogelino) [reply src/components/FacilityListItem.tsx:30]: I agree that topicLabels is clearer! 👍  Not too sure about issue though. Is being a family or a child an issue? 😛

Comment (vogelino) [reply src/components/FacilityListItem.tsx]: I renamed `semanticLabels` to `topicsLabels` and `targetLabels` to `targetAudienceLabels`

**Pull request merged**